src/app/common/hooks/account/use-old-wallet-salt.ts

```diff
@@ -0,0 +1,91 @@
+import { oldWalletSalt } from '@app/store/utils/vault-reducer-migration';
+import { decryptContent, encryptContent, getPublicKeyFromPrivate } from '@stacks/encryption';
+import { createWalletGaiaConfig } from '@stacks/wallet-sdk';
+import { useAsync } from 'react-async-hook';
+import { GaiaHubConfig, uploadToGaiaHub } from '@stacks/storage';
+import { fetchPrivate } from '@stacks/common';
+import { gaiaUrl } from '@shared/constants';
+import { useWalletState } from '@app/store/wallet/wallet.hooks';
+import { logger } from '@shared/logger';
+
+const oldWalletSaltFromGaia = 'old-wallet-salt-from-gaia';
```

Comment:
`salt-used-in-app-key-in-gaia` or something that better communicates what this is.  Let's avoid `old` as this isn't quite accurate.